utils/vqm2blif/src/base/vqm2blif_util.h

```diff
@@ -101,12 +101,46 @@ struct RamInfo {
     int port_a_addr_width = 0;
```

Comment:
If you know how to comment what this struct (and any other data types in here do) this is a good time to comment what each struct / data type is for.  If you only know some, comment them.